src/app/component/CartOverlay/CartOverlay.component.js

```diff
@@ -193,6 +193,24 @@ export class CartOverlay extends PureComponent {
         );
     }
 
+    renderCartAdditional() {
```

Comment:
'Additional' is not the best word to name this, it can be understood in too many ways.